clang/test/Frontend/amdgcn-machine-analysis-remarks.cl

```diff
@@ -2,12 +2,12 @@
 // RUN: %clang_cc1 -triple amdgcn-amd-amdhsa -target-cpu gfx908 -Rpass-analysis=kernel-resource-usage -S -O0 -verify %s -o /dev/null
 
 // expected-remark@+10 {{Function Name: foo}}
-// expected-remark@+9 {{    SGPRs: 13}}
-// expected-remark@+8 {{    VGPRs: 10}}
-// expected-remark@+7 {{    AGPRs: 12}}
-// expected-remark@+6 {{    ScratchSize [bytes/lane]: 0}}
+// expected-remark@+9 {{    SGPRs: foo.num_sgpr+(extrasgprs(foo.uses_vcc, foo.uses_flat_scratch, 1))}}
```

Comment:
This is another case that should have trivially resolved and produced a usable answer